code/src/java/pcgen/util/PJEP.java

```diff
@@ -301,7 +301,7 @@ else if (parent instanceof PlayerCharacter)
 	 * @return Returns the variableSource.
 	 */
 	@Deprecated
-	protected String getVariableSource()
+	private String getVariableSource()
```

Comment:
If you are going to make this private might as well delete this method. It seems deprecated anyway.